### PR TITLE
Add audit start button

### PR DESCRIPTION
**Pull request by theKHutDeveloper**
Added a 'Start audit' button that when clicked links to the first item in the items to audit list.

 [(1) Add 'Start audit of first item' to the 'My content' page](https://trello.com/c/jGGyfvjd/680-1-add-start-audit-of-first-item-to-the-my-content-page)

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -51,7 +51,7 @@ GEM
       public_suffix (>= 2.0.2, < 4.0)
     arel (8.0.0)
     ast (2.3.0)
-    autoprefixer-rails (7.2.1)
```
Comment (pmanrubia) [inline Gemfile.lock]: Not sure why all these gems are being updated.

File: spec/features/audit/lists/list_content_items_spec.rb [hunk 2]
```diff
@@ -32,6 +32,17 @@
     expect(page).to have_text("2 items")
   end
 
+  scenario "start audit button goes to first audit item" do
```
Comment (pmanrubia) [inline spec/features/audit/lists/list_content_items_spec.rb]: What about adding a test that checks the non-existence of the button when we have no content items?

**Comment by alecgibson** [comment]
[(1) Add 'Start audit of first item' to the 'My content' page](https://trello.com/c/jGGyfvjd/680-1-add-start-audit-of-first-item-to-the-my-content-page)

File: Gemfile.lock (hunk 1 above)
Comment (theKHutDeveloper) [reply Gemfile.lock]: @pmanrubia Would be happy to pair with you on this

**Pull request merged**